client/src/components/layouts/global.css

```diff
@@ -10,7 +10,7 @@ body {
 
 .btn-cta-big {
   max-height: 100%;
-  font-size: 1rem;
+  font-size: 1.3rem;
```

Comment:
Is the CTA on the landing the only CTA button on the whole website?

Reply:
only used in landing, intro, and profile when locked with viewed by a signed in user.
I update the breakpoint to make it look better.